src/main/java/net/coderbot/iris/pipeline/ShadowRenderer.java

```diff
@@ -685,8 +715,10 @@ public void renderShadows(LevelRendererAccessor levelRenderer, Camera playerCame
 	@Override
 	public void addDebugText(List<String> messages) {
 		messages.add("[Iris] Shadow Maps: " + debugStringOverall);
-		messages.add("[Iris] Shadow Distance: " + debugStringShadowDistance);
-		messages.add("[Iris] Shadow Culling: " + debugStringShadowCulling);
+		messages.add("[Iris] Shadow Distance: " + terrainFrustumHolder.getDistanceInfo());
+		messages.add("[Iris] Shadow Entity Distance: " + entityFrustumHolder.getDistanceInfo());
```

Comment:
This section of the debug screen is already excessively long...

Reply:
Can we do something like `Shadow culling T: <terrain> E: <entity>`

Reply:
That could work for now